ftw/simplelayout/aliasblock/contents/aliasblock.py

```diff
@@ -17,8 +17,15 @@ class AliasBlockSelectable(DefaultSelectable):
 
     def is_selectable(self):
         selectable = super(AliasBlockSelectable, self).is_selectable()
-        return selectable
+        is_sl_page = self.content.portal_type == 'ftw.simplelayout.ContentPage'
```

Comment:
Wouldn't it make more sense to check if the content provides `IContentPage` to make it possible to use objects which inherit from `ftw.simplelayout.ContentPage` without problems?

Reply:
@Nachtalb As of right now we only support exact this type. Other similar types like NewsFolder, or News may not work or may not make sense, since we don't show the archive portlet for example.